app/views/countries/partials/_citation.html.erb

```diff
@@ -0,0 +1,12 @@
+<section class="bg--navy-dark section-padding--bottom white">
+  <div class="container--small citation">
+    <p class="citation__title"><%= t('countries.shared.ancillary.citation_title'); %></p>
```

Comment:
I like to space out siblings - find it easier to read. No need to change here, but something to do going forward if you also find it easier.